src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -288,4 +288,22 @@ public function unescapeValue($value)
 
         return $value;
     }
+
+    /**
+     * @internal to be removed in 4.0
+     */
+    public static function normalizeName($name)
+    {
+        if (0 === strpos($name, 'env(') && ')' === substr($name, -1) && 'env()' !== $name) {
```

Comment:
@nicolas-grekas can you validate this approach? We dont want a deprecation for env(NAME) i guess =/